integration/env/common.go

```diff
@@ -3,57 +3,72 @@ package env
 import (
 	"fmt"
 	"os"
+	"strings"
 )
 
 const (
-	// EnvVarCircleCI is the process environment variable representing the
-	// CIRCLECI env var.
-	EnvVarCircleCI = "CIRCLECI"
-	// EnvVarCircleSHA is the process environment variable representing the
-	// CIRCLE_SHA1 env var.
-	EnvVarCircleSHA = "CIRCLE_SHA1"
-	// EnvVarKeepResources is the process environment variable representing the
-	// KEEP_RESOURCES env var.
-	EnvVarKeepResources = "KEEP_RESOURCES"
-	// EnvVarRegistryPullSecret is the process environment variable representing the
-	// REGISTRY_PULL_SECRET env var.
+	EnvVarCircleCI           = "CIRCLECI"
+	EnvVarCircleSHA          = "CIRCLE_SHA1"
+	EnvVarE2EKeepResources   = "E2E_KEEP_RESOURCES"
+	EnvVarE2ETestDir         = "E2E_TEST_DIR"
+	EnvVarGithubBotToken     = "GITHUB_BOT_TOKEN"
 	EnvVarRegistryPullSecret = "REGISTRY_PULL_SECRET"
 )
 
 var (
 	circleCI           string
 	circleSHA          string
+	githubToken        string
 	keepResources      string
 	registryPullSecret string
+	testDir            string
 )
 
 func init() {
 	circleCI = os.Getenv(EnvVarCircleCI)
-	keepResources = os.Getenv(EnvVarKeepResources)
+	keepResources = os.Getenv(EnvVarE2EKeepResources)
 
 	circleSHA = os.Getenv(EnvVarCircleSHA)
 	if circleSHA == "" {
 		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarCircleSHA))
 	}
 
+	testDir = os.Getenv(EnvVarE2ETestDir)
+	if testDir == "" {
+		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarE2ETestDir))
+	}
+
+	githubToken = os.Getenv(EnvVarGithubBotToken)
+	if githubToken == "" {
+		panic(fmt.Sprintf("env var %q must not be empty", EnvVarGithubBotToken))
+	}
+
 	registryPullSecret = os.Getenv(EnvVarRegistryPullSecret)
 	if registryPullSecret == "" {
 		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarRegistryPullSecret))
 	}
 }
 
-func CircleCI() string {
-	return circleCI
+func CircleCI() bool {
+	return circleCI == strings.ToLower("true")
```

Comment:
Shouldn't we rather use https://golang.org/pkg/strconv/#ParseBool? Also, `true` is already "to lower".

Reply:
Good call. I didn't know it exists. Also it should be `strings.ToLower(circleCI)` LoL.

Reply:
Problem with using strconv.ParseBool is it will fail on empty string and we need to handle an error.

Reply:
```suggestion
	return strings.ToLower(circleCI) =="true"
```